kong/error_test.go

```diff
@@ -49,3 +49,18 @@ func TestAPIError_Code(T *testing.T) {
 	require.True(ok)
 	assert.True(kongErr.Code() == 404)
 }
+
+func TestIsForbiddenErrE2E(T *testing.T) {
+	T.Skip()
```

Comment:
This test is expected to work if no license is present or if that's expired.

@pmalek do you have any tips to make this work in the go-kong CI (which always has a valid license)? Or, even better, any other option to trigger a 403 coming to mind?

Reply:
It might be a bit more involved but using a custom roundTripper like https://github.com/Kong/go-kong/blob/d8d2d9de7467a1870080a60b23cf059e24868bcf/kong/utils.go#L81-L103 which you can mock response of would fit here I guess.

`RunWhenEnterprise` unfortunately doesn't allow you to provide a custom client (where you'd use that roundTripper) but you could expand its parameters list 🤷.

Reply:
good point. I modified this with a custom client, thanks for the tip!